view/build.gradle

```diff
@@ -24,6 +24,7 @@ android {
 }
 
 dependencies {
+    compile "com.android.support:support-annotations:$support_version"
```

Comment:
I suggest to remove support annotations to avoid problems with versions.
In kotlin @LayoutRes and other annotation don't warning end user about wrong value.